evm-contracts/scripts/native_solc_compile_all

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+python3 -m pip install solc-select
+solc-select install 0.6.6
+solc-select install 0.7.6
```

Comment:
I think we need to add support for `0.8.0` now too.

Reply:
I added everything in the dev/ directory